### PR TITLE
Copter: Identify the message

### PR DESCRIPTION
**Pull request by muramura**
I am using ArduPilot on Multi Copter.
I want to understand the cause in the message.
I cannot easily find the cause in the same message.

### CODE REVIEW

File: ArduCopter/AP_Arming.cpp [hunk 1]
```diff
@@ -682,7 +682,7 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
             // in manual modes throttle must be at zero
             #if FRAME_CONFIG != HELI_FRAME
             if ((copter.flightmode->has_manual_throttle() || control_mode == Mode::Number::DRIFT) && copter.channel_throttle->get_control_in() > 0) {
-                check_failed(ARMING_CHECK_RC, true, "%s too high", rc_item);
+                check_failed(ARMING_CHECK_RC, true, "manual %s too high", rc_item);
```
Comment (khancyr) [inline ArduCopter/AP_Arming.cpp]: throttle ?

Comment (muramura) [reply ArduCopter/AP_Arming.cpp]: YES.

**Comment by rmackay9** [comment]
"manual throttle too high" is more clear than just "throttle too high"?  I'm not sure.. it's more words but not sure it's clearer..

File: ArduCopter/AP_Arming.cpp (hunk 1 above)
Comment (peterbarker) [reply ArduCopter/AP_Arming.cpp]: @khancyr ... or "collective" on a tradheli.

**Comment by rmackay9** [comment]
I'm not sure this is really any better so I think I'll close this.  Thanks anyway

**Pull request closed without merging**